package.json

```diff
@@ -41,6 +42,6 @@
     "typedoc": "^0.25.13",
     "typedoc-plugin-missing-exports": "^2.2.0",
     "typescript": "^5.3.3",
-    "typescript-eslint-language-service": "^5.0.5"
+    "typescript-eslint": "^8.0.0-alpha.13"
```

Comment:
I don't mind taking taking this (it's just a dev dependency), but is there any urgency driving us to upgrade to an alpha version?

Reply:
Just Dependabot churn, which could alternately be addressed with an ignore